cli/src/common.ts

```diff
@@ -292,28 +288,62 @@ export async function runPlatformHook(
   });
 }
 
-export function runCommand(command: string): Promise<string> {
-  return new Promise((resolve, reject) => {
-    exec(command, (error, stdout, stderr) => {
-      if (error) {
-        reject(stdout + stderr);
-      } else {
-        resolve(stdout);
-      }
-    });
-  });
+export interface RunCommandOptions {
+  cwd?: string;
+}
+
+export async function runCommand(
+  command: string,
+  args: readonly string[],
+  options: RunCommandOptions = {},
+): Promise<string> {
+  const p = new Subprocess(command, args, options);
+
+  try {
+    return await p.output();
+  } catch (e) {
+    if (e instanceof SubprocessError) {
+      // old behavior of just throwing the stdout/stderr strings
+      throw e.output ? e.output : e.code;
+    }
+
+    throw e;
+  }
+}
+
+export async function runNativeRun(
+  args: readonly string[],
+  options: RunCommandOptions = {},
+): Promise<string> {
+  try {
+    return await runCommand('native-run', args, options);
+  } catch (e) {
+    if (e === ERROR_COMMAND_NOT_FOUND) {
+      logFatal(
+        `${c.input('native-run')} not found in PATH\n` +
+          `Are you sure ${c.strong('native-run')} is installed?`,
```

Comment:
would be good to give some insight about native-run, install from where? or how?